chainer/datasets/fashion_mnist.py

```diff
@@ -8,6 +8,23 @@
 from chainer.datasets._mnist_helper import preprocess_mnist
 
 
+def fashion_mnist_label(label):
```

Comment:
* How about naming it `get_fashion_mnist_labels` and return all labels as list?
* Need to expose this under `chainer/datasets/__init__.py`.